Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -837,7 +837,7 @@ public void onError(final RequestError error) {
       final boolean ok = runInPooledThreadAndWait(new Runnable() {
         @Override
         public void run() {
-          server.edit_format(filePath, selectionOffset, selectionLength, consumer);
+          server.edit_format(filePath, selectionOffset, selectionLength, -1, consumer);
```

Comment:
OK

My initial goal was just to keep status quo here.
Later we may start providing the real line length.